src/tasks/validate-package.js

```diff
@@ -46,6 +47,9 @@ module.exports = function validatePackage(dir, functionHandler, restApiModule) {
 						throw routeMessage + 'requests custom headers but does not enumerate any headers';
 					}
 				}
+				if (methodConfig.success && !validHttpCode(methodConfig.success)) {
```

Comment:
this is strange, it will fail on `success: { contentType: 'text/plain' }`. Why did you need this?